packages/core/src/Cardano/types/Genesis.ts

```diff
@@ -1,22 +1,19 @@
-import { Epoch, Int64, NetworkMagic, UInt64 } from '@cardano-ogmios/schema';
 import { Lovelace } from '.';
-import { NetworkId } from '../NetworkId';
-
-export type Ratio = number;
+import { NetworkId, NetworkMagic } from '../NetworkId';
 
 /**
- * A compact (without genesis UTxO) representation of the genesis configuration.
+ * A compact (without genesis UTxO) representation of the genesis configunumbern.
  */
 export interface CompactGenesis {
   systemStart: Date;
   networkMagic: NetworkMagic;
   networkId: NetworkId;
-  activeSlotsCoefficient: Ratio;
-  securityParameter: UInt64;
-  epochLength: Epoch;
-  slotsPerKesPeriod: UInt64;
-  maxKesEvolutions: UInt64;
-  slotLength: Int64;
-  updateQuorum: UInt64;
+  activeSlotsCoefficient: number;
+  securityParameter: bigint;
```

Comment:
Some of these properties being typed as `bigint` caught my attention this time around. I now recall the reason Ogmios WSP schema has them as `UInt64`, however it's not specified to be `bigint` in the generated [TypeScript definition](https://github.com/CardanoSolutions/ogmios/blob/d16e4ede0d40d0b4d40f2fb47f265513942d845f/clients/TypeScript/packages/schema/src/index.ts#L102). 

https://github.com/CardanoSolutions/ogmios/blob/c45d1a1a65d7f2fa35a0d60be25fe2baf0f58172/architectural-decisions/accepted/013-typescript-client_bigint-parsing.md

 https://github.com/CardanoSolutions/ogmios/commit/0a91b15e6a761819e78f7e74d603104b6ea3a174